cmd/snap-preseed/main.go

```diff
@@ -48,6 +48,7 @@ first-boot startup time`
 
 type options struct {
 	Reset               bool   `long:"reset"`
+	ResetReexeced       bool   `long:"reset-reexeced"`
```

Comment:
we usually use env var to manage reexec, an option work too, I wonder if it should be marked hidden though, is it is not for direct user consumption. Maybe also we should call it --reset-chroot ?